back-office/advanced-parameters/database/advanced_parameters-database-sql_manager.md

```diff
@@ -30,21 +30,84 @@ At first, the SQL Manager page displays 2 messages:
+
+1) Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are keeped.
+
+2) Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+3) Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+#### Bulk actions
+
+Clicking on the _Bulk actions_ button will give the user the ability to perform bulk actions for selected sql queries (using the check boxes). The user can:
+- Select all/Unselect all: This button will select/unselect all check boxes next to each row on the table.
+- Delete selected: This button will delete the selected sql queries. 
+After clicking on "Delete selected", a modal with title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462))
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the selected webservice keys are deleted, a successful message is displayed "The selection has been successfully deleted."
+
+#### Columns
+
+The page displays the SQL Manager list with the following columns:
+
+1) ID
+
+It's possible to sort by id asc or desc and search by id.
 
+2) SQL query name
+
+It's possible to sort by SQL query name asc or desc and search by SQL query name.
+
+3) SQL query 
+
+It's possible to sort by SQL query asc or desc and search by SQL query.
+
+4) Actions 
+
+- View. Executes the query, and get the results displayed in an HTML table, right within the PrestaShop interface. This is the main action and it can be performed by clicking on the line.
 When the user views the SQL request with no result, a warning is displayed: "This SQL query has no result."
+- Export. Executes the query, and downloaded the result in CSV format.
 - Edit. The user can edit a query as often as necessary, to refine it and get better results.
 - Delete. Once a query is not used anymore (or simply because it does not work), you can delete it by clicking this button and confirming your choice.
+After clicking on _Delete_, a modal with title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" 
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the message is deleted and a successful message is displayed "Successful deletion."
+
+#### Multistore behavior
+
+As SQL query can't be associated to specific shop(s) during the creation, so all the SQL query from all stores are listed, no matter the context selected.
```

Comment:
I'm not sure I understand: a SQL query cannot be dedicated to a specific shop? By default, it affects all shops?

Reply:
Yes, it affects all shops, you can't create an SQK query just for one shop

Reply:
```suggestion
By default, a SQL query affects all shops. Since it can't be associated with any specific shop, all the SQL queries listed on this page are used in an all shops context.
```